src/config/layers.json

```diff
@@ -1065,5 +1065,41 @@
         "color": "#0868ac"
       }
     ]
+  },
```

Comment:
If this is specific to Australia, let's make sure we ave a new config for Australia